contracts/bridge/x/v4/RegistrationV4.sol

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Immutable Pty Ltd 2018 - 2024
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {CoreV4} from "./CoreV4.sol";
+
+contract RegistrationV4 {
+    CoreV4 public immutable imx;
+
+    constructor(address payable _imx) {
+        imx = CoreV4(_imx);
+    }
+
+    function getVersion() external view returns (string memory) {
```

Comment:
The ordering of functions should be external & public transaction functions, then external & public view functions, then external & public pure functions, then internal, the private.  
That is, put this external view function after the external and public non-view functions.

Reply:
@drinkcoffee, I'll fix it up, but is this a well-known pattern?